homeassistant/components/opower/config_flow.py

```diff
@@ -36,6 +38,7 @@ async def _validate_login(
         login_data[CONF_UTILITY],
         login_data[CONF_USERNAME],
         login_data[CONF_PASSWORD],
+        login_data.get(CONF_TOTP_SECRET, None),
```

Comment:
`None` is the default value returned if the key is missing in the dict. We can remove the second parameter.